src/kernel/include/fpu.h

```diff
@@ -39,7 +38,7 @@ static inline void          fpu_set(int);
 static inline int           fpu_get_info(void);
 static inline int           fpu_check_fxsr(void);
 static inline int           fpu_check_sse(void);
-
+#include "thd.h"
```

Comment:
I think it would be better if you put it along with other includes. Since they are not in different micro.

Reply:
Put thd.h along with other includes can cause "no implicit reference" error.

Reply:
@huachuan Can you provide the full error here (if you put thd.h up with the other includes) along with context about what you think is happening, so that we can understand what's happening?

Reply:
The problem in fpu.h file is connected with the one thd.h file. If thd.h is with other includes in fpu.h file. I will get 
warning: implicit declaration of function 'fpu_thread_init(thd)' [-Wimplicit-function-declaration]
same as fpu_switch(thd) functions and so on.
warning: nested extern declaration of 'fpu_thread_init'